docs/pages/en-us/Getting-Started.md

```diff
@@ -81,7 +79,7 @@ docker-compose -f docker-compose.yml -f docker-compose-debug.yml up -d
 
 ### Submitting Files
 
-The compose file will by default look in a directory './input', relative to the compose file directory, for files that should be submitted to data processing. The results of data processing will by default be output to a folder './output', also relative to the compose file location. These locations can be controlled via the `INPUT_DOCUMENTS_LOCAL_FOLDER` and `OUTPUT_DOCUMENTS_LOCAL_FOLDER` environment variables. Details on how the files are output can be found in the repository for the task-receiver service [here](https://github.com/CAFDataProcessing/data-processing-service/tree/develop/utils/data-processing-task-receiver#task-message-output). Any files present in the input directory on launch of the services will be submitted and any files added to the folder later will also be submitted.
+The compose file will by default look in a directory './input', relative to the compose file directory, for files that should be submitted to data processing. The input location can be controlled via the `INPUT_DOCUMENTS_LOCAL_FOLDER` environment variable. The results of data processing will by default be indexed into an Elasticsearch instance stood up by the compose file. If a failure occurs during at the workflow worker the failure will be output to a folder './output' releative to the compose file directory. Any files present in the input directory on launch of the services will be submitted and any files added to the folder later will also be submitted.
```

Comment:
"during at the workflow worker" doesn't sound right.

Reply:
Reworded.